dashboard/app/models/levels/level.rb

```diff
@@ -477,6 +477,14 @@ def summary_for_lesson_plans
     summary
   end
 
+  def additional_reference_links
+    links = {
+      mapReference: map_reference,
+      referenceLinks: reference_links
+    }
+    links
```

Comment:
`links` is unneeded, I believe - can be expressed as
```rb
def additional_reference_links
  {
    mapReference: map_reference,
    referenceLinks: reference_links
  }
end
```
On a related note, why is this needed? Seems like this could be easily handled in levels_helper.rb where a mapping is occurring anyway.